packages/babel-parser/test/fixtures/es2017/async-call/parenthesized-argument-object-double-proto/input.js

```diff
@@ -0,0 +1 @@
+async({ __proto__: x, __proto__: y })
```

Comment:
Maybe you could also add a test like
```js
async ({ __proto__: x, __proto__: y }) => {}
```
which is valid and shouldn't report an error.